docs/bugs_and_glitches.md

```diff
@@ -175,24 +175,38 @@ This bug existed for all battles in Gold and Silver, and was only fixed for sing
 ```diff
 -	; BUG: 1/256 chance to fail even for a 100% effect chance,
 -	; since carry is not set if BattleRandom == [hl] == 255
+- 	call BattleRandom
 +	ld a, [hl]
-+	cp 100 percent
-+	jr z, .ok
- 	call BattleRandom
++	sub 100 percent
++	; If chance was 100%, RNG won't be called (carry not set)
++	; Thus chance will be subtracted from 0, guaranteeing a carry
++	call c, BattleRandom
  	cp [hl]
--	pop hl
--	ret c
-+	jr c, .ok
+	pop hl
```

Comment:
Watch the indentation.

Reply:
The indentation [is present](https://github.com/pret/pokecrystal/blob/add58fb00d73495655243c4cf13b730bbc1b3242/docs/bugs_and_glitches.md#moves-with-a-100-secondary-effect-chance-will-not-trigger-it-in-1256-uses), this looks like GitHub's preview is broken.
The tabs are also present in [my fork's version](https://github.com/ISSOtm/pokecrystal/blob/patch-1/docs/bugs_and_glitches.md#moves-with-a-100-secondary-effect-chance-will-not-trigger-it-in-1256-uses) of the file.

Reply:
It has to have a space preceeding the tab, due to how `diff` files work, and to be consistent with the rest of this document.

Reply:
Ah, sorry, I wasn't aware of that. I just pushed a commit that fixes that issue.